.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+If possible, please include the following information in a pastebin such as `pastebin.osuosl.org` when submitting an issue:
```

Comment:
The pastebin is good for logs and `gem list` only. Everything else can remain in the issue text.

Actually, I think they can also attach the log files to an issue. That or pastebin, anything is preferable over pasting things in the issue.